pandas/core/arrays/_mixins.py

```diff
@@ -278,8 +278,12 @@ def fillna(
 
         if mask.any():
             if method is not None:
-                func = missing.get_fill_func(method)
-                new_values, _ = func(self._ndarray.copy(), limit=limit, mask=mask)
+                # TODO: check value is None
```

Comment:
already done in validate_fillna_kwargs

Reply:
thanks, will update (possibly in upcoming CLN PR to avoid clogging the CI right now)